eras/conway/impl/src/Cardano/Ledger/Conway/Rules/Enact.hs

```diff
@@ -64,15 +63,18 @@ enactmentTransition = do
       where
         newPP = ensPParams st `applyPPUpdates` ppup
     HardForkInitiation pv -> pure $ st {ensProtVer = pv}
-    TreasuryWithdrawals wdrl -> do
-      let newWdrls = foldr' (<>) (Coin 0) $ ensWithdrawals st
```

Comment:
Note for reviewer: Previously it was checking the total of already enacted withdrawals, rather than the ones that currently being proposed. It should have been:
```haskell
let newWdrls = foldr' (<>) (Coin 0) wdrls
```

Reply:
That's a very good catch!